Sources/AppcuesKit/Presentation/DeepLinkHandler.swift

```diff
@@ -46,8 +46,8 @@ internal class DeepLinkHandler: DeepLinkHandling {
                 self = .debugger(destination: DebugDestination(pathToken: pathTokens[safe: 1]))
             } else if pathTokens.count == 2, pathTokens[0] == "verify" {
                 self = .verifyInstall(id: pathTokens[1])
```

Comment:
Side note, a query param seems "more right" for the verify deeplink check too. I might update that at some point.